velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -339,6 +350,36 @@ TEST_P(PrestoSerializerTest, dictionaryWithExtraNulls) {
   testRoundTrip(dictionary);
 }
 
+// TODO: re-enable after we support dictionary encoding in Presto serializer.
```

Comment:
Sorry, I'm probably missing something obvious, but how are these tests different from what's already covered in PrestoSerializerTest.encodings?

Reply:
Ah, I see. But the tests call flushEncoded which is only used by test but not production code.